src/backend/app.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const dbRouter = require("./routes/database");
+const bodyParser = require("body-parser");
+
+const app = express();
+
+app.use(express.json());
+app.use(bodyParser.json({ type: ["application/json", "application/fhir+json"] }));
+app.use(express.static(`${__dirname}/../../public/`));
```

Comment:
the app will host the static files in the public directory.  Due to the way the app works, this causes a couple quirks when you run the dev version.  Since webpack is watching the frontend files for changes, and `nodemon` is watching the `/public` directory for changes, when the app first starts the backend app loads and goes up, but it takes extra time for webpack to build the files.

When webpack finishes building the files, it puts them in the `/public` directory (or updates the existing ones) and then the backend app reloads.

In the production version of the app, we run these commands in sequence instead of in parallel, since webpack only has to build the files and finish, instead of building the files and watching them indefinitely.